polars/polars-core/Cargo.toml

```diff
@@ -156,6 +156,7 @@ chrono-tz = { version = "0.8", optional = true }
 comfy-table = { version = "6.1.4", optional = true, default_features = false }
 either.workspace = true
 hashbrown.workspace = true
+hyperloglogplus = { version = "0.4.1", features = ["const-loop"] }
```

Comment:
Can we make this an optional dependency? It should also reside in `polars-ops`, we could feature gate this under `approx_unique`.

Reply:
> Can we make this an optional dependency? It should also reside in `polars-ops`, we could feature gate this under `approx_unique`.

Done, but since I had some questions, I have left the dependency in the core temporarily.

Reply:
done.